spec/models/retirement_manager_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe RetirementManager do
+  describe "#check" do
+    let(:ems) { FactoryGirl.create(:ext_management_system) }
+    let(:zone) { FactoryGirl.create(:zone, :name => "api_zone", :ext_management_systems => [ems]) }
+    let(:my_server) { FactoryGirl.create(:miq_server, :zone => zone) }
+
+    it "check runs" do
+      allow(MiqServer).to receive(:my_server).and_return(my_server)
```

Comment:
Why `allow` rather than `expect`?

When I read rspec, my brain translates:
`allow` - This may or may not happen, I'm either not sure or can't find a better way to write this test.
`expect` - I expect this thing to happen (or not to happen)